tests/object/test.bats

```diff
@@ -0,0 +1,84 @@
+# PURPOSE:
+# I will test the following commands: object create, get, list, update, url, delete, and clean.
+load ../test_setup.bash
+
+# constants
+obj1="test1.txt"
+obj2="test2.txt"
+obj3="test3.txt"
+
+@test "object create" { # object create test
+  
```

Comment:
Can we remove any blank lines at the start and end of the test blocks?

Reply:
Sure, will do.